src/components/CustomTable/style.js

```diff
@@ -43,7 +46,9 @@ export const DeleteProgress = styled(CircularProgress)`
   margin-top: 10px;
 `;
 export const EditDeletCell = styled(TableCell)`
-  display: flex;
+  @media (max-width: 768px) {

```

Comment:
I Think thats ok, but look out for themes file and I guess we have defined the breakpoints there used them instead .. So that there will be consistency throughout the app..